Foundation/NSURL.swift

```diff
@@ -29,10 +29,14 @@ internal let kCFURLPlatformPathStyle = kCFURLPOSIXPathStyle
 #endif
 
 private func _standardizedPath(_ path: String) -> String {
+#if os(Windows)
```

Comment:
Hmm, could you add a comment why the absolute path check is not needed on windows.